docs/porting/psa/spm.md

```diff
@@ -248,6 +272,12 @@ Processor access    |Secure RAM        |Secure FLASH|Nonsecure RAM      |Nonsecu
 `Secure Write`      |   V              |    V       |        V          |    ?
 `Secure Execute`    |   X?             |    V       |        X          |    ?
 
+
+###  TF-M SPM porting for (ARMv8-M targets)
+
+TF-M HAL functions are defined in `tfm_spm_hal.h`
```

Comment:
Let's link to this once the Doxygen merges. Also, is there more to add to this section? It feels sparse.

Reply:
i'm not sure why it is not generated